### PR TITLE
Add postman sprite

### PR DESCRIPTION
**Pull request by owalk**
This adds the initial use case of level prompts and subsequently fixes the level prompt code so that it now works.

### CODE REVIEW

File: game/HUD.py [hunk 1]
```diff
@@ -77,6 +82,9 @@ def __init__(self):
         self._level_hint_added_flag: bool = False
         self._level_hint_text: str = ""
         self._level_paused: bool = False
+        self._level_hint_txt_spr = pygame.sprite.Sprite()
+        self._level_box_spr = pygame.sprite.Sprite()
```
Comment (Sean-Nishi) [inline game/HUD.py:86]:
```suggestion
        self._level_hint_text_sprite = pygame.sprite.Sprite()
        self._level_box_sprite = pygame.sprite.Sprite()
```

Comment (Sean-Nishi) [reply game/HUD.py:86]: Convention so far has been to avoid shorthand.

File: game/tiles/entities/characters/NPCs/NPC.py [hunk 2]
```diff
@@ -88,6 +90,8 @@ class NPC(Character):
         Set player for NPC to track
     collided_with_player(self)
         NPC actions on collision with player
+    neuteral_collided_with_player(self)
```
Comment (Sean-Nishi) [inline game/tiles/entities/characters/NPCs/NPC.py:93]: Spelling of "neutral"

File: game/tiles/entities/characters/NPCs/NPC.py [hunk 3]
```diff
@@ -446,6 +452,14 @@ def collided_with_player(self) -> None:
             elif current_player_action == Actions.throw:
                 self.set_state_thrown()
 
+    def neuteral_collided_with_player(self):
```
Comment (Sean-Nishi) [inline game/tiles/entities/characters/NPCs/NPC.py:455]: Spelling of "neutral"

File: game/tiles/entities/characters/NPCs/NPC.py [hunk 4]
```diff
@@ -446,6 +452,14 @@ def collided_with_player(self) -> None:
...
+        """Player has collided with a neuteral sprite."""
+        self._hud.start_prompt_timer()
+        child_class: str = self.__class__.__name__
+
+        prompt_text = prompt_strings.npc_prompts[child_class]["hint_text"]
```
Comment (Sean-Nishi) [inline game/tiles/entities/characters/NPCs/NPC.py:460]: need type hint

**Review by Sean-Nishi — CHANGES_REQUESTED** [review]
Only a few comments

File: game/HUD.py (hunk 1 above)
Comment (owalk) [reply game/HUD.py:86]: fixed, need a new commit for the name change

**Review by Sean-Nishi — APPROVED** [review]

**Pull request merged**